manager/app/build.gradle.kts

```diff
@@ -23,8 +23,8 @@ android {
 
     buildTypes {
         release {
-            isMinifyEnabled = true
-            isShrinkResources = true
+            isMinifyEnabled = false
```

Comment:
please don't touch this

Reply:
ok let me fix it i forgot to change this after test